src/utils/RegHubUtil.js

```diff
@@ -117,59 +121,97 @@ module.exports = {
     repositoryServerActions.reposLoading({repos: []});
 
     hubUtil.request({
-      url: `${REGHUB2_ENDPOINT}/namespaces/`,
+      url: `${REGHUB2_ENDPOINT}/namespaces/`
```

Comment:
To confirm - I'll remove the namespaces call, use only the orgs obtained and append the username to the list?